recipe/meta.yaml

```diff
@@ -15,26 +15,24 @@ package:
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 8897491277ce80a1ea49306552be778ece4d0bf8fad3373388936554c32bccd8
+  sha256: 193055dbee8701ad62d5471c5287b092ce53bb77f7f810ec68bf96816f6acaa2
 
 build:
   number: 0
   script: {{ PYTHON }} -m pip install . -vv
-  skip: True  # [py<36]
+  skip: True  # [py<38]
   skip: True  # [linux and (s390x or ppc64le)]
 
 requirements:
   host:
-    - python
+    - python >=3.6
     - pip
-    - setuptools
-    - wheel
```

Comment:
Mainly, they are required for python packages
```
    - setuptools
    - wheel
```
They are implicitly already added to the host env but make sure to add them explicitly in the host section, see https://github.com/anaconda-distribution/perseverance-skills/blob/main/sections/Reference/Recipe_standards.md#the-requirements-section

Reply:
yep, I suggest keeping the recipe standards open and referring to them often as you make changes until you become familiar :)

Reply:
thanks for the tip!